modules/installation-azure-config-yaml.adoc

```diff
@@ -52,9 +56,13 @@ compute: <2>
   platform:
     azure:
       type: Standard_D2s_v3
+      encryptionAtHost: true
       osDisk:
         diskSizeGB: 512 <5>
         diskType: Standard_LRS
+        diskEncryptionSet:
+          resourceGroup: disk_encryption_set_resource_group
+          name: disk_encryption_set_name
       zones: <6>
       - "1"
```

Comment:
There is an optional subscriptionId field that can be used here for a disk encryption set in another subscription.

Reply:
I have added this field to the configuration parameters table and the sample yaml file. PTAL

Reply:
Just to be clear, this field is optional. If it is not specified, the same subscriptionId used for the install will be used here. This is only required when the disk encryption set is in another subscription.